docs/libvirt-pod-networking.md

```diff
@@ -0,0 +1,135 @@
+# Using CNI Interfaces and libvirt for networking
+
+Author: Fabian Deutsch \<fabiand@redhat.com\>
+
+## Introduction
+
+Today networking in KubeVirt is not integrated with Kubernetes, a user can
+rather choose to create a specific VM Spec definition to create a particular
+libvirt domxml, to reuse an existing physical device.
+
+Today networking in KubeVirt is not integrated with Kubernetes. In order to
```

Comment:
It's a little jarring for two consecutive paragraphs to start with the exact same words: "Today networking in KubeVirt is not integrated with Kubernetes"... Is it possible to re-word one of them?